ang/civicase/CaseCard.js

```diff
@@ -15,12 +15,22 @@
   });
 
   module.controller('CivicaseCaseCardController', function ($scope, getActivityFeedUrl, DateHelper) {
+    $scope.data.selected = $scope.data.selected || false;
```

Comment:
If `$scope.data.selected` is not true, doe not that mean its already false? Why is this line required?

Reply:
We needed this to initialise the variable as it was throwing some error for some condition. which I can't recall right now. However , removing this line don't show any impact on the functionality. So removing it.